test/fixtures/TypeScriptImports.svelte

```diff
@@ -0,0 +1,109 @@
+<script lang="ts">
```

Comment:
Is there a reason for this fixture to be this big?

Reply:
It was my decision, because I wanted to (manually) ensure that all features are taken into account and that the sourcemap is well handled.

Reply:
Maybe it could be reduced now...

Reply:
Finally, I think that we should keep this fixture intact, it is very useful to reproduce edge cases.